### PR TITLE
Creating README.md with minor introductions

### PR DESCRIPTION
**Pull request by AnirbanBanik1998**
Added a readme.md file

### CODE REVIEW

**Comment by ghoshbishakh** [comment]
@AnirbanBanik1998 please mention that the prerequisites are for building the app from the source code not playing the game. And please remove the "===============" from the last line

Thanks!! 😄

**Comment by AnirbanBanik1998** [comment]
Oh I'm really sorry will try to improve upon this as soon as possible

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+freemex
+--------
+Online trading is the act of placing buy/sell orders for financial securities and/or currencies with the use of a brokerage's internet-based proprietary trading platforms. ... 
+Stocks, bonds, options, futures and currencies can all be traded online.
+Freemex-In this project you have to contribute in building the app from the source code. 
+
+Prerequisites
+--------------
```
Comment (ghoshbishakh) [inline README.md]: Prequisites for building the source code not playing the game. So make a heading 'Building from source' and then put the subheading prerequisites

File: README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+freemex
+--------
+Online trading is the act of placing buy/sell orders for financial securities and/or currencies with the use of a brokerage's internet-based proprietary trading platforms. ... 
+Stocks, bonds, options, futures and currencies can all be traded online.
+Freemex-In this project you have to contribute in building the app from the source code.
```
Comment (ghoshbishakh) [inline README.md]: 'Freemex-In this project you have to contribute in building the app from the source code. ' remove this line

**Review by ghoshbishakh — CHANGES_REQUESTED** [review]

**Comment by AnirbanBanik1998** [comment]
Made the changes

File: README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+freemex
+--------
+                                 Building from source
```
Comment (ghoshbishakh) [inline README.md]: Please see how markdown works. In github you can see a preview of the markdown file.
This is not proper syntax bringing int to the right by appending some spaces.

use single # for h1
use double ## for h2 .. like that.

See:
https://daringfireball.net/projects/markdown/syntax

**Review by ghoshbishakh — APPROVED** [review]

**Comment by ghoshbishakh** [comment]
Plsease do the fixes and I will merge. Thanks!

**Comment by ghoshbishakh** [comment]
@AnirbanBanik1998 thanks!
But please check the preview yourself:
https://github.com/AnirbanBanik1998/freemex/blob/ba36f5a532e2cf075d41f0837e0b83cfefc2dc7b/README.md

Type the markdown in the github comments markdown editor and press preview and you will see.

'#'              -> creates a header

---------- '  -> also creates a header

So use any one of them

I prefre #

**Comment by AnirbanBanik1998** [comment]
I think now it is without much error.Sorry for taking so long

**Comment by ghoshbishakh** [comment]
@AnirbanBanik1998 solid work!

**Comment by AnirbanBanik1998** [comment]
Thanks!

**Pull request merged**